modules/appeals_api/config/schemas/v2/10182.json

```diff
@@ -3,56 +3,6 @@
   "description": "JSON Schema for VA Form 10182",
   "$ref": "#/definitions/nodCreate",
   "definitions": {
-    "address": {
```

Comment:
While this is the direction we eventually want to end up, I don't think we can actually make this change right now since our existing controllers use this file to determine their validations and as what gets returned from our `#schema` actions.

Removing these definitions would be a breaking API change, so we may need to have two sets of schema files for a while - one that is for the existing endpoints, and another with the shared schema updates for the newly namespaced endpoints. Let's discuss options on how we want to approach this issue with the whole team.